src/core/state.go

```diff
@@ -895,7 +818,7 @@ func (state *BuildState) queueResolvedTarget(target *BuildTarget, dependent Buil
 
 // queueTarget enqueues a target's dependencies and the target itself once they are done.
 func (state *BuildState) queueTargetAsync(target *BuildTarget, dependent BuildLabel, rescan, forceBuild, forSubinclude, building bool) {
-	defer state.taskDone(true, false)
+	defer state.taskDone(true)
```

Comment:
A little confused by these "sythetic" tasks.

Reply:
Yeah, it just keeps the numbers in the display from going bonkers when we generate all these additional tasks